common/src/cams/cases.ts

```diff
@@ -98,14 +98,32 @@ export type CaseNoteInput = {
   createdOn?: string;
 };
 
-export type DxtrCase = CaseSummary & {
+type ClosedDismissedReopened = {
   closedDate?: string;
   dismissedDate?: string;
   reopenedDate?: string;
 };
 
+export type DxtrCase = CaseSummary & ClosedDismissedReopened;
+
 export type SyncedCase = DxtrCase &
   Auditable & {
     documentType: 'SYNCED_CASE';
     id?: string;
   };
+
+export function isCaseClosed<T extends ClosedDismissedReopened>(bCase: T) {
+  const { closedDate, reopenedDate } = bCase;
+  return closedDate ? (reopenedDate ? closedDate >= reopenedDate : true) : false;
+}
+
+export function isCaseOpen<T extends ClosedDismissedReopened>(bCase: T) {
+  return !isCaseClosed(bCase);
+}
+
+export function getCaseIdParts(caseId: string) {
+  const parts = caseId.split('-');
+  const divisionCode = parts[0];
+  const caseNumber = `${parts[1]}-${parts[2]}`;
+  return { divisionCode, caseNumber };
+}
```

Comment:
Should this perform any sanity checking? For example, if it is passed something without any dashes or not enough, it doesn't throw, but returns `undefined` where `parts[n]` does not exist.